instructions.md

```diff
@@ -35,11 +35,11 @@
 **For every bounding box, include one of the following labels:**
 1. **[Car or Van or SUV](#car-or-van-or-suv)**: Vehicle designed primarily for personal use, e.g. sedans, hatch-backs, wagons, vans, mini-vans, SUVs and jeeps.   
 
-2. **[Truck](#truck)**: Vehicles primarily designed to haul cargo including lorrys, trucks. 
+2. **[Truck](#truck)**: Vehicles primarily designed to haul cargo including pick-ups, lorrys, trucks and semi-tractors. Trailers hauled after a semi-tractor should be labeled as vehicle.trailer.
```

Comment:
Use "Trailer" here instead of vehicle.trailer

Reply:
Done